### PR TITLE
Updated information about exclusion/inclusion from FLoC calculation

### PR DESCRIPTION
**Pull request by samdutton**
As discussed here: WICG/floc/issues/85.

Also updated here: GoogleChrome/developer.chrome.com/pull/615.

### CODE REVIEW

File: src/site/content/en/blog/floc/index.md [hunk 1]
```diff
@@ -257,10 +257,16 @@ greater-than-typical number of people who visit sites in a sensitive category, t
 removed. Negative financial status and mental health are among the sensitive categories covered by 
 this analysis. 
 
-Websites opt out of FLoC by setting the [Permissions-Policy](https://www.w3.org/TR/permissions-policy-1/#introduction) 
-header `interest-cohort=()`. For the [FLoC origin trial](https://developer.chrome.com/origintrials/#/view_trial/213920982300098561) 
-in Chrome 89, websites that don't opt out will be included in the FLoC calculation if Chrome detects 
-that they are sites [which load ads or ads-related resources](https://github.com/WICG/floc/issues/82). ([Ad Tagging in Chromium](https://chromium.googlesource.com/chromium/src/+/master/docs/ad_tagging.md) explains how Chrome's ad detection mechanism works.) Websites can also, of course, simply not access or record the cohort IDs of their visitors.
+Websites can exclude a page from the FLoC calculation by setting a [Permissions-Policy](https://www.w3.org/TR/permissions-policy-1/#introduction) header `interest-cohort=()` for that page. For pages that haven't
```
Comment (jpmedley) [inline src/site/content/en/blog/floc/index.md:260]: Is `interest-cohort` part of the FLoC project? I can't find this is implemented or specified.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/floc/index.md (hunk 1 above)
Comment (samdutton) [reply src/site/content/en/blog/floc/index.md:260]: > Is `interest-cohort` part of the FLoC project? I can't find this is implemented or specified.

Yes: [github.com/WICG/floc#opting-out-of-computation](https://github.com/WICG/floc#opting-out-of-computation).

Comment (jpmedley) [reply src/site/content/en/blog/floc/index.md:260]: Have you the header elsewhere in the article? If you haven't, you should, or you can link to the section in the explainer.

Comment (samdutton) [reply src/site/content/en/blog/floc/index.md]: I've added a link.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**